00_Base/src/config/sub/ServerConfigUtilMessageBrokerAmqp.ts

```diff
@@ -0,0 +1,19 @@
+import { IsNotEmpty, IsString } from 'class-validator';
+import { Service } from 'typedi';
+import 'reflect-metadata';
+
+@Service()
+export class ServerConfigUtilMessageBrokerAmqp {
+  @IsString()
+  @IsNotEmpty()
+  url: string;
+
+  @IsString()
+  @IsNotEmpty()
+  exchange: string;
+
+  constructor() {
```

Comment:
Please note: constructors in these `config` related classes are intended only to be defaults giving us the option to be able to eliminate the `local` version of the config which can be chosen as the "default"